config/src/main/java/org/springframework/security/config/annotation/web/configurers/openid/OpenIDLoginConfigurer.java

```diff
@@ -443,6 +491,16 @@ public AttributeConfigurer attribute(String name) {
 			private boolean required = false;
 			private String type;
 
+			/**
+			 * Creates a new instance named "default-attribute".
+			 * The name can by updated using {@link #name(String)}.
+			 *
+			 * @see AttributeExchangeConfigurer#attribute(String)
+			 */
+			private AttributeConfigurer() {
+				this.name = "default-attribute";
```

Comment:
The default attribute name is "default-attribute". This is primarily to avoid a `NullPointerException`, users should override the name in the `name()` setter.